### ISSUE TITLE
Create a page to view the full spec

### ISSUE THREAD

**Issue by danirabbit** [hughsie/oars#6]
Much like the linked page to the TIGRS spec, it would be useful to have a page where we can see the whole spec in one place. At the moment it seems like you have to use the [generator](https://github.com/hughsie/oars/blob/master/docs/generate.html) to see what is included in OARS.

It would be nice to have this in a human read-able format somewhere in this repo. Especially it would make it easy to open new issues in GitHub linking to specific lines or to be able to submit pull requests

**Comment by hughsie** [comment]
I think it needs to be something like wayland, where you can take the machine readable spec and generate some beautiful docs, and *also* generate the generator if you see what I mean. Something XMLy would be easy to parse for me.

**Comment by danirabbit** [comment]
Yeah I think that makes sense. I was almost going to suggest a yaml or json or something too. The exact file format probably doesn't matter too much :)
